shared/oae/api/oae.api.util.js

```diff
@@ -1628,6 +1628,17 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
     var redirect = exports.redirect = function() {
 
         /**
+         * Redirect the current user to the same page on the specified tenant
+         *
+         * @param  {Tenant}     tenant  The tenant to which to redirect the user
+         */
+        var tenant = function(tenant) {
+            var relative = url().attr('relative');
+            var target = window.location.protocol + '//' + tenant.host + relative;
```

Comment:
I think you can just do `var target = '//' + tenant.host + relative;` with the same result